charts/cf-runtime/values.yaml

```
@@ -648,6 +648,14 @@ runtime:
     workflowLimits:
       # -- Maximum time allowed to the engine to wait for the pre-steps (aka "Initializing Process") to succeed; seconds.
       MAXIMUM_ALLOWED_TIME_BEFORE_PRE_STEPS_SUCCESS: 600
+
+      # -- Maximum allowed time for post-steps to finish (optional!).
+      # Overrides POST_STEPS_GRACE_PERIOD_MIN environment variable in cf-api for this specific runtime only.
+      # It's maximum time allowed to wait for the post-steps to finish before killing engine; seconds.
```

```suggestion
      # It's maximum time allowed to wait for the post-steps to finish before terminating build; seconds.
```

+ I would rephrase this doc line for more clarity, unlikely end user knows what post-step is

---

@masontikhonov as we already expose environmental variable named `POST_STEPS_GRACE_PERIOD_MIN` to onprem customers I think it's fine to keep this terminology to avoid confusion.